src/site/markdown/server/manage-webadmin.md

```diff
@@ -2515,3 +2516,35 @@ Response codes:
 
  - 200: A list of corresponding tasks is returned
  - 400: Invalid status value
+
+## Cassandra extra operations
+
+Some webadmin features to manage some extra operations on Cassandra tables, like solving inconsistencies on projection tables.
+Such inconsistencies can be for example created by a fail of the DAO to add a mapping into 'mappings_sources`, while it was successful
+regarding the `rrt` table.
+
+ - [Operations on mappings sources](#Operations_on_mappings_sources)
+
+### Operations on mappings sources
+
+You can do a series of action on `mappings_sources` projection table :
+
+```
+curl -XPOST /cassandra/mappings?action=[ACTION]
+```
+
+Will return the taskId corresponding to the related task. Actions supported so far are :
+
+ - SolveInconsistencies : cleans up `mappings_sources` projection table and repopulate it correctly.
```

Comment:
This is done by first cleaning all the content of the Mapping sources index and then repopulating it. In the meantime, listing sources of a mapping might be leading to inconsistencies.